src/twisted/internet/iocpreactor/tcp.py

```diff
@@ -359,7 +359,7 @@ def __init__(self, sock, protocol, clientAddr, serverAddr, sessionno, reactor):
         self.sessionno = sessionno
         logPrefix = self._getLogPrefix(self.protocol)
         self.logstr = f"{logPrefix},{sessionno},{self.clientAddr.host}"
-        self.repstr = "<{} #{} on {}>".format(
+        self.repstr: str = "<{} #{} on {}>".format(
```

Comment:
worrying that this wasn't hit by tests :(

Reply:
oh it was hit just took a while for the coverage check to integrate the 3.11 iocp windows job